src/ragger/conftest/base_conftest.py

```diff
@@ -0,0 +1,171 @@
+import pytest
+from typing import Optional
+from pathlib import Path
+from ragger.firmware import Firmware
+from ragger.backend import SpeculosBackend, LedgerCommBackend, LedgerWalletBackend
+from ragger.navigator import NanoNavigator
+from ragger.utils import app_path_from_app_name
+
+from ragger.conftest import configuration
+
+BACKENDS = ["speculos", "ledgercomm", "ledgerwallet"]
+
+DEVICES = ["nanos", "nanox", "nanosp", "all"]
+
+FIRMWARES = [Firmware('nanos', '2.1'),
+             Firmware('nanox', '2.0.2'),
+             Firmware('nanosp', '1.0.3')]
+
+
+def pytest_addoption(parser):
```

Comment:
On app-u2f, I've added more options: https://github.com/LedgerHQ/app-u2f/blob/develop/tests/speculos/conftest.py#L30-L31
How can it be handled?

Reply:
I also have a client fixture, but this could be added on the app conftest

Reply:
As agreed on slack, let's keep them in mind for a v2